### DISCUSSION TITLE
Cannot find plugin

### DISCUSSION THREAD

**Discussion by GPana** [neuropoly/axondeepseg#377] [Help needed] [answered]
Hello all, 

I have followed the steps to download AxonDeepSeg.
I have arrived to the point where I am instructed to load plugin ads_plugin.py

However, I cannot find the plugin in my computer anywhere.

Any turnarounds to this?

Thx in advance

George

**Comment by Stoyan-I-A** [comment]
Hello George,

Note: Currently, if you want to use the Graphical User Interface (GUI), you have to install the development version of AxonDeepSeg.

If you are on Mac, after you use the clone command `git clone https://github.com/neuropoly/axondeepseg.git` (which you've already done if you followed the documentation for the development version installation), you can enter `open .` in the terminal and it should open the finder interface. The plugin can be found inside the axondeepseg folder.

If you are on Linux, it should be in home/axondeepseg.



Let us know if you found it.

**Comment by mariehbourget** [comment, marked as answer]
Hi @GPana, thank you for reaching out!

Could you please let us know which OS and version you are using?

I’m assuming these (https://axondeepseg.readthedocs.io/en/stable/documentation.html#installation) are the steps you followed to install AxonDeepSeg and FSLeyes, however I just want to double check with you.

Which version of AxonDeepSeg did you installed?

If you installed the Stable release (https://axondeepseg.readthedocs.io/en/stable/documentation.html#stable-release), you won’t be able to use FSLeyes.

To use FSLeyes, you have to install the development version (https://axondeepseg.readthedocs.io/en/stable/documentation.html#development-version) via git clone.

When installed with the development version, the `ads_plugin.py` file will be in the cloned repository (folder `axondeepseg` in your home). 

You can find a video describing the installation of AxonDeepSeg and FSLeyes [here](https://www.youtube.com/watch?v=dz2LqQ5LpIo&ab_channel=AxonDeepSeg).

Let us know if you have any questions and we’ll be happy to help you out.

**Comment by GPana** [comment]
Thx for your help!

**Comment by jcohenadad** [comment]
@GPana please let us know if the problem is solved on your end, and what solved it. This will be useful to know in case this happens to another user. Thank you!
